app/config/sources.inc.php

```diff
@@ -579,462 +49,1041 @@
+        ],
+        'supported_locales' => [
+            'az', 'cs', 'cy', 'de', 'dsb', 'en-GB', 'es-AR', 'es-CL', 'es-ES',
+            'es-MX', 'fa', 'fr', 'hsb', 'it', 'kab', 'ja', 'ko', 'pl', 'pt-BR',
+            'ro', 'ru', 'sq', 'sr', 'uk', 'zh-CN', 'zh-TW',
+        ],
+    ],
+    'teach/smarton/surveillance.lang' => [
+        'flags' => [
+            'opt-in' => ['all'],
+        ],
+        'supported_locales' => [
+            'az', 'cs', 'cy', 'de', 'dsb', 'en-GB', 'es-AR', 'es-CL', 'es-ES',
+            'es-MX', 'fa', 'fr', 'hsb', 'it', 'ja', 'kab', 'ko', 'pl', 'pt-BR',
+            'ro', 'ru', 'sq', 'sr', 'uk', 'zh-CN', 'zh-TW',
+        ],
+    ],
+    'teach/smarton/tracking.lang' => [
+        'flags' => [
+            'opt-in' => ['all'],
+        ],
+        'supported_locales' => [
+            'az', 'cs', 'cy', 'de', 'dsb', 'en-GB', 'es-AR', 'es-CL', 'es-ES',
+            'es-MX', 'fa', 'fr', 'hsb', 'it', 'ja', 'kab', 'ko', 'pl', 'pt-BR',
+            'ro', 'ru', 'sq', 'sr', 'uk', 'zh-CN', 'zh-TW',
+        ],
+    ],
+    'thunderbird/channel.lang' => [
+        'flags' => [
+            'opt-in' => [
+                'ar', 'ast', 'az', 'bg', 'bn-BD', 'br', 'ca', 'cs', 'cy', 'da',
+                'de', 'dsb', 'el', 'en-GB', 'es-AR', 'es-ES', 'et', 'eu', 'fi',
+                'fr', 'fy-NL', 'ga-IE', 'gd', 'gl', 'he', 'hr', 'hsb', 'hu',
+                'hy-AM', 'id', 'is', 'it', 'ja', 'kab', 'ko', 'lt', 'nb-NO',
+                'nl', 'nn-NO', 'pa-IN', 'pl', 'pt-BR', 'pt-PT', 'rm', 'ro',
+                'ru', 'si', 'sk', 'sl', 'sq', 'sr', 'sv-SE', 'tr', 'uk', 'vi',
                 'zh-CN', 'zh-TW',
             ],
-        'thunderbird/channel.lang' =>
-            [
-                'cy', 'cs', 'de', 'en-GB', 'es-ES', 'fi', 'fr', 'it', 'ja', 'kab', 'ko',
-                'lt', 'nl', 'pl', 'pt-BR', 'ru', 'sq', 'sv-SE', 'uk', 'zh-TW',
-            ],
-        'thunderbird/features.lang'      => $thunderbird_locales,
-        'thunderbird/index.lang'         => $thunderbird_locales,
-        'thunderbird/start/release.lang' => $thunderbird_locales,
+        ],
+        'supported_locales' => [
+            'cy', 'cs', 'de', 'en-GB', 'es-ES', 'fi', 'fr', 'it', 'ja', 'kab',
+            'ko', 'lt', 'nl', 'pl', 'pt-BR', 'ru', 'sq', 'uk', 'zh-TW',
+        ],
+    ],
+    'thunderbird/features.lang' => [
+        'deadline'          => '2017-02-20',
+        'supported_locales' => $thunderbird_locales,
+    ],
+    'thunderbird/index.lang' => [
+        'supported_locales' => $thunderbird_locales,
     ],
+    'thunderbird/start/release.lang' => [
+        'deadline'          => '2016-08-01',
+        'supported_locales' => $thunderbird_locales,
+    ],
+];
 
-    'about:healthreport' =>
-    [
-        'fhr.lang' => array_diff($firefox_desktop_android, $mozorg_locales),
+$firefoxhealthreport_lang = [
+    'fhr.lang' => [
+        'supported_locales' => array_diff($firefox_desktop_android, $mozorg_locales),
     ],
+];
 
```

Comment:
Interesting, I did add them, but apparently the last commit didn't make it to GH.